solidity/dashboard/src/services/tbtc-authorization.service.js

```diff
@@ -270,25 +256,34 @@ const fetchDelegationInfo = async (web3Context, operatorAddress) => {
   )
 }
 
-const fetchCreatedBondsEvents = async (
+const fetchCreatedBonds = async (
   web3Context,
   operatorAddresses,
   sortitionPoolAddress
 ) => {
-  return contractService.getPastEvents(
-    web3Context,
-    KEEP_BONDING_CONTRACT_NAME,
-    "BondCreated",
-    {
-      fromBlock: CONTRACT_DEPLOY_BLOCK_NUMBER[KEEP_BONDING_CONTRACT_NAME],
-      filter: {
-        operator: operatorAddresses,
-        sortitionPool: sortitionPoolAddress,
-      },
+  return (
+    await contractService.getPastEvents(
+      web3Context,
+      KEEP_BONDING_CONTRACT_NAME,
+      "BondCreated",
+      {
+        fromBlock: CONTRACT_DEPLOY_BLOCK_NUMBER[KEEP_BONDING_CONTRACT_NAME],
+        filter: {
+          operator: operatorAddresses,
+          sortitionPool: sortitionPoolAddress,
+        },
+      }
+    )
+  ).map((_) => {
+    return {
+      operator: _.returnValues.operator,
+      holder: _.returnValues.holder,
+      referenceID: _.returnValues.referenceID,
     }
-  )
+  })
 }
 
+// TODO: add later when fetching bonding section
 const fetchBondReassignedEvents = async (
```

Comment:
Let's comment out this function due to `fetchBondReassignedEvents is assigned a value but never used` warning.

Reply:
Actually, I'd rather just remove it for now. Copied & paste in the [issue](https://github.com/keep-network/keep-core/issues/1793) as an example what should be done later.